src/test/java/com/microsoft/azure/relay/SendReceiveTest.java

```diff
@@ -73,6 +74,16 @@ public void websocketSmallSendSmallResponseTest() {
 		clientTask.join();
 	}
 
+	@Test
+	public void websocketSendTextResponseTextTest() {
+		CompletableFuture<Void> listenerTask = sendAndReceiveTextWithWebsocketListener(WS_MSG, WS_MSG);
```

Comment:
I would like to see larger data (>64kb) be tested as well. Maybe you can create small and large strings out of the `SMALL_BYTES` and `LARGE_BYTES` from above.

Reply:
>64kb is not possible in a single message. The `DefaultMaxTextMessageBufferSize` is by default `RelayConstants.DEFAULT_CONNECTION_BUFFER_SIZE`, which is 64Kb. I added a test with a string of 64KB.